components/ScheduleSection.js

```diff
@@ -0,0 +1,242 @@
+import SectionTitle from './SectionTitle'
+
+const sessions = [
+  {
+    id: 1,
+    type: 'FLASH',
+    time: '10:00 - 10:10',
+    sessionFirstTopic: 'Welcome Speech by Venue'
+  },
+  {
+    id: 2,
+    type: 'MAIN',
+    time: '10:30 - 11:15',
+    sessionFirstTopic: 'Let\'s start width React in the right way',
+    sessionFirstAvatar: 'https://avatars2.githubusercontent.com/u/25113?v=3&s=400',
+    sessionFirstSpeaker: 'Somkiat Puisungnoen (somkiat.cc)',
+    sessionSecondTopic: 'Scalable React app Architecture',
+    sessionSecondAvatar: 'https://avatars2.githubusercontent.com/u/134488?v=3&s=460',
+    sessionSecondSpeaker: 'Zack Siri (Founder at Artellectual)'
+  },
+  {
+    id: 3,
+    type: 'MAIN',
+    time: '11:15 - 12:00',
+    sessionFirstTopic: 'Live coding / mob programming session',
+    sessionFirstAvatar: 'https://avatars1.githubusercontent.com/u/193136?v=3&s=400',
+    sessionFirstSpeaker: 'Thai Pangsakulyanont (Task World)',
+    sessionSecondTopic: 'Behavior Driven Development in React Native',
+    sessionSecondAvatar: 'https://avatars1.githubusercontent.com/u/19698625?v=3&s=400',
+    sessionSecondSpeaker: 'Tanakorn numrubporn (CTO at Planforfit)'
```

Comment:
Having multiple fields that starts with same prefix is a code smell (I learned to recognize this pattern from [Sandi Metz talk, all the little things](https://www.youtube.com/watch?v=8bZh5LMaSmE), I recommend watching it!)

It suggests that something could be extracted. For example,

```js
  {
    id: 3,
    type: 'MAIN',
    time: '11:15 - 12:00',
    sessions: [
      {
        topic: 'Live coding / mob programming session',
        speakerName: 'Thai Pangsakulyanont',
        speakerTitle: 'Taskworld'
      },
      {
        topic: 'Behavior Driven Development in React Native',
        speakerName: 'Tanakorn Numrubporn',
        speakerTitle: 'CTO at Planforfit'
      }
    ]
  }
```

Note in previous comment, we can generate avatar URL from speaker’s name, that’s why I separated speaker name and title into different fields.